start

```diff
@@ -24,45 +24,26 @@ show_help() {
 
 while :; do
   case $1 in    
-    -d|--data-only)
-      sfd="false"
-      data="true"
-      receivers="true"
-      ;;
-    -r|--receivers-disabled)
-      receivers="false"
-      ;;  
     -h|--help)   # Call a "show_help" function to display a synopsis, then exit.
       show_help
       exit
       ;;
     *)
       break
   esac
-
   shift
 done
 
 cd "${projectRoot}"
 
-if [ "${sfd}" = "true" ]; then
-  ./fcp-fd-data/scripts/start -d;
-  echo "Waiting for cosmosDB to start"
-  ./fcp-fd-auth/scripts/start -d;
-  ./fcp-fd-landing-page/scripts/start -d;
-  ./fcp-fd-messages/scripts/start -d;
-  ./fcp-fd-messages-processor/scripts/start -d;
-  ./fcp-fd-file-receiver/scripts/start -d;
-  ./fcp-fd-file-processor/scripts/start -d;
-  ./fcp-fd-probe/scripts/start -d;
-fi
-
-if [ "${receivers}" = "true" ]; then
-  ./fcp-fd-customer-receiver-messages/scripts/start -d;
-fi
-
-if [ "${data}" = "true" ]; then
-  ./fcp-fd-data/scripts/start -d;
-  echo "Waiting for cosmosDB to start"
```

Comment:
Again removed references to Cosmos since we're not using it yet. When we do implement it we can copy it over from `ffc-sfd-core` as previously mentioned.